bed/methods.go

```diff
@@ -9,18 +9,23 @@ import (
 // Current methods satisfy requirements for the following interfaces:
 // bed.BedLike
 
+// GetChrom grabs the chrom of the bed struct
```

Comment:
changing "grabs" to "returns" may help people who are starting out with coding by using more precise (and google-able) language.